src/dgraph.ts

```diff
@@ -32,11 +32,11 @@ async function dqlQuery(query: string, variables: Record<string, any> = {}): Pro
   return response.json();
 }
 
-async function dqlMutate(mutate: string): Promise<GraphQLResponse> {
+async function dqlMutate(mutate: string | any): Promise<GraphQLResponse> {
```

Comment:
```suggestion
async function dqlMutate(mutate: string | Object): Promise<GraphQLResponse> {
```
I am not really a TS expert, but AFAIK, for JSON, the input will always be an object (and string for RDF). So, should the type be explicitly set to `Object` here (or `Record<string, any>`)? Because I am not sure what all can `any` allow and if users can face any glitches with that.

Reply:
Because you're not calling any methods on it, it won't matter if you use **any** or **object**: https://stackoverflow.com/a/18961904/271450 - It is just really important to check whether or not it is a string

Reply:
okay! then maybe switch the `typeof` statement to use string:
```
typeof mutate === 'string' ? "application/rdf" : "application/json"
```
That way we can be sure rdf is sent only when it is string and not when it could have been some non-object (like int).

Once you make this and the above suggested change, then it should be good to go in master.